lib/pbench/test/unit/server/test_file_tree.py

```diff
@@ -0,0 +1,177 @@
+from pathlib import Path
+import re
+
+import hashlib
+
+import pytest
+import shutil
```

Comment:
I think hashlib and shutil are part of the standard library, so they should both be in the first group.

Reply:
I don't mind (and actually kind of like) splitting the "project" from the "other" imports, but splitting "standard" and "third-party" bothered me initially.  Now, having tangled with trying to manage dependencies, I'm starting to appreciate the distinction, although, perhaps not yet enough to maintain three groups....